### DISCUSSION TITLE
how to use direciton function

### DISCUSSION THREAD

**Discussion by guhao1129** [inocan-group/vue3-google-map#225] [Q&A]
google map javascript api have direction，StreetView and Places functions, how can i use this function with this compants?
thankyou

**Comment by HusamElbashir** [comment]
Hi @guhao1129 

We don't have convenience apis for these functions. You might be able to hook them up yourself using some of the patterns here:
https://github.com/inocan-group/vue3-google-map#advanced-usage

If you need to use another library for these features specifically you can load the Google Maps javascript API externally and pass a prop so as to not load the script twice. See this for an example:
https://github.com/inocan-group/vue3-google-map/issues/99#issuecomment-1237395931
